README.md

```diff
@@ -60,10 +63,53 @@ The bundle will deploy:
 - 1 `coordinator` unit (`tempo-coordinator-k8s` charm)
```

Comment:
perhaps I would write the app names in this listing too, separate from their scale, so the user will know what to expect to see in `juju status`.
How about:

```
The bundle will deploy:

- a `tempo` application with 1 unit (`tempo-coordinator-k8s` charm)
- a `tempo-worker` application with 3 units (`tempo-worker-k8s` charm), configured with `role=all`
- an `s3-integrator` application with 1 unit (`s3-integrator` charm)
```